src/Equinor.ProCoSys.IPO.WebApi/Startup.cs

```diff
@@ -207,6 +208,18 @@ public void ConfigureServices(IServiceCollection services)
                 services.AddSingleton<IPcsBusSender>(new DisabledServiceBusSender());
             }
             services.AddHostedService<VerifyApplicationExistsAsPerson>();
+
+            Smtp smtp = new Smtp()
```

Comment:
this is not the way to initiate from configuration
See all the xxxOptions classes and setup/use of these
![image](https://github.com/equinor/procosys-invitation-for-punch-out-api/assets/19545528/e36a3b54-22b7-45a4-81b2-880f674453bf)

Your Smtp class should be named SmtpOptions